mega-linter-runner/generators/mega-linter/templates/mega-linter.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         id: ml
 
         # All available variables are described in documentation
-        # https://megalinter.io/latest/config-file/
+        # https://megalinter.io/latest/configuration/
```

Comment:
@nvuillam One of the conflicts was that this link was changed. I checked and the https://megalinter.io/latest/configuration/ contains all the sections from the side navigation, and https://megalinter.io/latest/config-file/ is one of those sections.

https://megalinter.io/latest/configuration/ does not seem clickable from the left navigation



Where/when this changed?

![image](https://github.com/oxsecurity/megalinter/assets/27212526/5a73cccd-1be5-438c-9385-a0413ae8c895)
![image](https://github.com/oxsecurity/megalinter/assets/27212526/65a3a5a2-ae4c-4f13-8ac3-6801cf97703a)

Reply:
@echoix I splitted configuration into sub menus , but i let the old page for SEO ^^
I should delete it in a next version :)

Reply:
Could that page be replaced with a page that links to the subsections ? That url is prettier than the new ones, and it _was_ used. At least something that redirects to one of the current pages

Reply:
@echoix please be my guest to do so :)